### PR TITLE
Added images, revisions to the problems section and a new challenges …

### PR DESCRIPTION
**Pull request by bmanuel-ginger**
…section.

### What's in this PR?

A few final edits to the flutter article to expand the challenges section, problems section and the addition of an image.

### CODE REVIEW

File: _posts/2020-04-11-native-mobile-to-flutter.markdown [hunk 1]
```diff
@@ -18,25 +20,26 @@ Over the last six months, the Ginger mobile team has been busy rewriting our nat
...
 ### What problems were we hoping to solve?
 
-We had two working native mobile applications, but lacked feature parity for Android and iOS users.
+We are too small to have dedicated native iOS and Android apps.  Maintaining redundancy across two mobile teams would have required us to maintain four to six developers rather than as few as two to three in a cross platform environment.
 
-Android and iOS development took place at different paces, with varying quality and cost.
+The organizational complexity of maintaining iOS and Android teams created inefficiencies from engineering all the way up to marketing.  We hypothesized that having a single team and product would reduce the social complexity within our team, allowing us to move faster and have a more unified product experience.
 
-Maintaining redundancy across two mobile teams would have required us to maintain four to six developers rather than as few as two to three in a cross platform environment.
+Finally, Our legacy native apps had accumulated years of technical debt; we were ready for a full rewrite.
```
Comment (aboisvert) [inline _posts/2020-04-11-native-mobile-to-flutter.markdown]: Typo:  remove capitalized "O" in "Our"

**Review by aboisvert — APPROVED** [review]
:sparkle: :+1:

File: _posts/2020-04-11-native-mobile-to-flutter.markdown [hunk 2]
```diff
@@ -51,7 +54,7 @@ _Regardless of our choice our entire team would have to..._
 3. add sufficient test coverage to confidently launch the application
 4. deliver a highly secure application. 
 
-At the end of the day, the developers called it.  Flutter felt much more approachable and less like web a web platform than React Native.
+At the end of the day, the developers called it.  Flutter felt much more approachable and less like a web platform than React Native.
```
Comment (jcarver989) [inline _posts/2020-04-11-native-mobile-to-flutter.markdown:57]: Looks like a double space? `it.  Flutter`

**Review by jcarver989 — APPROVED** [review]
Looks good!

**Pull request merged**